modules/nf-core/modules/ensemblvep/main.nf

```diff
@@ -16,32 +16,36 @@ process ENSEMBLVEP {
     path  extra_files
 
     output:
-    tuple val(meta), path("*.ann.vcf"), emit: vcf
-    path "*.summary.html"             , emit: report
-    path "versions.yml"               , emit: versions
+    tuple val(meta), path("*.ann.vcf")  , optional:true, emit: vcf
```

Comment:
Looks like the module should be updated in nf-core/modules to allow for more optional outputs :)

Reply:
my bad 🙈